index.js

```diff
@@ -35,27 +37,46 @@ const defaultOptions = {
   runtime: false,
   corejs: false,
   debug: false,
-  ignoreBrowserslistConfig: true,
+  ignoreBrowserslistConfig: false,
   browsers: defaultBrowsers,
+  include: [
+    // XXX: Workaround for babel/babel#8019.
+    'transform-classes',
+  ],
+  exclude: [
+    // Seems to be added by default with minimum benefit.
+    'web.dom.iterable',
+  ],
 };
 
+function getTargets(
+  env,
+  { target, targets, configPath, ignoreBrowserslistConfig },
+) {
+  if (env !== 'production') {
+    return target === 'node' ? { node: 'current' } : { esmodules: true };
+  }
+  if (target === 'node') return { node: '10.0' };
```

Comment:
bleh we should distinguish "app" build targets from "lib" ones, but that can be later

Reply:
though a TODO would be appropriate